test/helpers/index.ts

```diff
@@ -239,18 +239,26 @@ export const createBatchMintData = async ({
   listNow = true,
   projectId = 1_234_567_890,
   scheduleStartTime,
+  holdbackPercentage = 0,
 }: {
   hre: CustomHardHatRuntimeEnvironment;
   fifoMarket: FIFOMarket;
   listNow?: boolean;
   projectId?: number;
   scheduleStartTime?: number;
+  holdbackPercentage?: number;
 }): Promise<Parameters<Removal['mintBatch']>[3]> => {
   const actualScheduleStartTime =
     scheduleStartTime ?? (await getLatestBlockTime({ hre }));
   const packedData = hre.ethers.utils.defaultAbiCoder.encode(
-    ['uint256', 'uint256', 'address', 'bool'],
-    [projectId, actualScheduleStartTime, fifoMarket.address, listNow]
+    ['uint256', 'uint256', 'uint256', 'address', 'bool'],
```

Comment:
heads up that this syntax exists and could make life easier going forward

https://forum.openzeppelin.com/t/erc20-batchtransfer-functionality/6978/6

Reply:
hmm, do you mean the use of `encodeABI()` (which seems to be a web3 function)? or using multicall? (which i'm not seeing how it fits into this step)

Reply:
Ethers equivalent is `encodeFunctionData` I believe ([docs](https://docs.ethers.io/v5/api/utils/abi/interface/)), but you're right that it wouldn't be useful since I forgot we just accept a generic data param. Given that minting isn't part of the base erc 1155 spec, I think we should probably consider just dropping the mint inheritance like we did for rNORI and just accept a struct as the parameter rather than decoding it on-chain. Would clean up our code a lot and allow us to use objects instead of encoding ([example](https://ethereum.stackexchange.com/questions/118280/how-to-call-a-contract-function-from-reactjs-that-takes-a-struct-as-an-argument))

Reply:
(not in the scope of this PR, just a comment for now -- maybe add a todo though?)

Reply:
oh nice... yeah it would definitely be cool to be able to work with structs/objects directly instead of encoded data.  not the first time you've recommended removing the inheritance of "mint"